src/components/TodoForm/TodoForm.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useState } from 'react';
+import usersFromServer from '../../api/users';
+import { Todo } from '../../types/Todo';
+
+interface TodoFormProps {
+  onAddTodo: (todo: Todo) => void;
+  maxId: number;
+}
+
+export const TodoForm: React.FC<TodoFormProps> = ({ onAddTodo, maxId }) => {
+  const [todo, setTodo] = useState({
+    id: 0,
+    title: '',
+    completed: false,
+    userId: 0,
+  });
+
+  const [isUserChosen, setIsUserChosen] = useState(true);
+  const [isTitleEntered, setIsTitleEntered] = useState(true);
+
+  function handleInputChange(event: React.ChangeEvent<HTMLInputElement>) {
+    setTodo(prevTodo => ({
+      ...prevTodo,
+      title: event.target.value,
+      id: maxId + 1,
+    }));
+
+    setIsTitleEntered(true);
+  }
+
+  function handleSelectChange(event: React.ChangeEvent<HTMLSelectElement>) {
```

Comment:
It is important to follow the same way of creating functions everywhere
```suggestion
  const handleSelectChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
```